db/src/db.rs

```diff
@@ -2087,4 +2087,127 @@ mod tests {
         // dangling, if we give a :db/id explicitly.
         assert_transact!(conn, "[{:test/dangling {:db/id \"t\" :test/many 11}}]");
     }
+
+    #[test]
+    fn test_explode_reversed_notation() {
+        let mut conn = TestConn::default();
+
+        // Start by installing a few attributes.
+        assert_transact!(conn, "[[:db/add 111 :db/ident :test/many]
+                                 [:db/add 111 :db/valueType :db.type/long]
+                                 [:db/add 111 :db/cardinality :db.cardinality/many]
+                                 [:db/add 222 :db/ident :test/component]
+                                 [:db/add 222 :db/isComponent true]
+                                 [:db/add 222 :db/valueType :db.type/ref]
+                                 [:db/add 333 :db/ident :test/unique]
+                                 [:db/add 333 :db/unique :db.unique/identity]
+                                 [:db/add 333 :db/index true]
+                                 [:db/add 333 :db/valueType :db.type/long]
+                                 [:db/add 444 :db/ident :test/dangling]
+                                 [:db/add 444 :db/valueType :db.type/ref]]");
+
+        // Check that we can explode direct reversed notation, entids.
+        let report = assert_transact!(conn, "[[:db/add 100 :test/_dangling 200]]");
```

Comment:
For completeness, can we have an example of reversed and non-reversed use of an attribute in the same tx?

```
[[:db/add 100 :test/_dangling 200]
 [:db/add 200 :test/dangling 101]]
```

or something like that.